### PR TITLE
Adds info windows and text to page describing map content.

### PR DESCRIPTION
**Pull request by meagancheese**
Also makes it so that only one info window can be open at a time to avoid clutter.

### CODE REVIEW

**Review by GrantComm — APPROVED** [review]
LGTM

**Review by GrantComm — APPROVED** [review]
LGTM

**Review by jalexanderqed — APPROVED** [review]

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -208,12 +208,35 @@ function checkLogin() {
   });
 }
 
+let markers = [];
```
Comment (acmcarther) [inline portfolio/src/main/webapp/script.js:211]: This is a "global variable" which isn't great, but with the current structure of the application it isn't real clear how to fix that.

One option I have in mind makes it a bit better, but might be onerous, so [optional].

Create a class "Map", with constructor accepting `map` and with functions:
* `createMarker(latLng, windowText)`: creates the marker and infowindow and stores them locally.
* `clearMarkers()`: Erases all markers and closes windows.

**Review by acmcarther — COMMENTED** [review]
LG, but I'd like to re-review if you refactor `markers` and `infoWindows` into a class. LMK what you want to do.

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (acmcarther) [reply portfolio/src/main/webapp/script.js:211]: (after creating the class, create a singleton instance of the Map and assign to  global variable when you're first creating the map)

**Pull request merged**